README.md

```diff
@@ -249,13 +218,119 @@ Invoking the `eval` method of each dataset yields all metrics reported in the pa
 ```
 Most `eval` methods take in predicted labels for `all_y_pred` by default, but the default inputs vary across datasets and are documented in the `eval` docstrings of the corresponding dataset class.
 
+## Using the example scripts
+In `examples/`, we provide a set of scripts that can be used to train models on the WILDS datasets.
+The scripts are configured to default to the models and hyperparameters from the papers.
+Baseline results can thus be easily replicated with commands like:
```

Comment:
Unfortunately, this is no longer true for the unlabeled data baselines (including the updated ERM numbers), because we do random search for them. So I think we should probably change this to something along the lines of the defaults are set reasonably, so you can just run them, and for the exact hyperparameters, please check CodaLab?